Chp9Lsn11/Chp9Lsn11/Program.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using UserInputHandler;
+
+namespace Chp9Lsn11
+{
+    public class Program
+    {
+        static void Main(string[] args)
+        {
+            string answer = "Error";
+            string userCommand = TextMenu(answer);
+            MultipleType FinalAnswer = CallOption(userCommand);
+            PrintAnswer(userCommand, FinalAnswer);
+            Console.ReadLine();
+        }
+
+        public static string TextMenu(string answer)
+        {
+            bool resolved = false;
+            
+            Console.WriteLine("This program has three options. Option 'reverse' allows you to enter a series of numbers ");
+            Console.WriteLine("that are returned in reverse order. Option 'average' calculates the average of a series of");
+            Console.WriteLine("numbers. Option 'equation' calculates x of the linear equation. Type the name of the option");
+            Console.WriteLine("you desire and press enter.");
+            Console.WriteLine("");
+
+            if(answer != "Error")
+            {
+                resolved = true;
+            }
+
+            while (resolved == false)
+            {
+                string input = Console.ReadLine();
+
+                if (input != "reverse" && input != "average" && input != "equation")
+                {
+                    Console.WriteLine("");
+                    Console.Write("Invalid Input!");
+                }
+                else
+                {
+                    answer = input;
+                    resolved = true;
+                }
+            }
+
+            return answer;
+        }
+
+        public static MultipleType CallOption(string optionSelect)
+        {
+            MultipleType userInput = new MultipleType();
+
+            if (optionSelect == "reverse")
+            {
+                userInput.integer = UserInput.UserString();
+                userInput.integer = ComputerCalculate(userInput.integer);
```

Comment:
You have three different `ComputerCalculate` methods that are distinguished simply by the type of parameter each takes.  This is called "function overloading" and is a legitimate technique.  However, in this case the design is self-limiting (that means it has a problem).  If somebody asks you to add another feature through another `ComputerCalculate` which takes an integer, you will be stuck because you already have a `ComputerCalculate` that takes an integer.

This suggests that the method name `ComputerCalculate` should not be "overloaded."  Instead the name should be unique for each option.  This way it won't matter what the parameters are.